src/vmm/src/devices/virtio/balloon/device.rs

```diff
@@ -401,9 +401,12 @@ impl Balloon {
         Ok(())
     }
 
-    pub(crate) fn signal_used_queue(&self) -> Result<(), BalloonError> {
+    pub(crate) fn signal_used_queue(&self, qidx: usize) -> Result<(), BalloonError> {
         self.interrupt_trigger()
-            .trigger_irq(IrqType::Vring)
+            .trigger(VirtioInterruptType::Queue(
+                qidx.try_into()
+                    .unwrap_or_else(|_| panic!("balloon: invalid queue id: {qidx}")),
```

Comment:
Let's change `qidx` to be `u16` and remove this ugliness. Or add a the conversion as a first line:
```rust
let idx: u16 = qidx.try_into().expect("Invalid queue index");
```

Reply:
If I change `qidx` to be `u16` here I'd need to do the conversion in all calling sites.

Reply:
Ok, what about at least doing a conversion on a separate line and replacing `.unwrap_or_else(|_| panic!(...))` with `expect(&format!("..."))`?

Reply:
the `expect(&format!("..."))` triggers clippy. That's how I had it at the beginning.

Reply:
As discussed offline, we can fix this later.